### DISCUSSION TITLE
Unlink Project

### DISCUSSION THREAD

**Discussion by shadowyoonji** [vercel/vercel#5827] [Help] [answered]
I'm quite new to using vercel so I apologise if this is obvious, I have deployed a couple of projects using vercel but I see that in the new update it favours project linking, I can't figure out how to launch my new project as a separate project instead of overriding my last one. I see the --name/--n function has been disabled and I can't find the supposed .vercel directory to delete to avoid this as it stated in the guide. Any ideas? 

I'm using node.js and I'm on a mac

**Comment by styfle** [comment, marked as answer]
If you want to link your current directory to a Vercel Project, you can run `vercel link`.

If the current directory is already linked to an existing project, `vercel link` will work regardless 👍
